### PR TITLE
#2187 - Ensure Field Charts Dynamic Metrics Connection Field Counts Are Correct

### PR DESCRIPTION
**Pull request by kodinkat**
fixes: #2187

### CODE REVIEW

File: dt-metrics/counters/counter-post-stats.php [hunk 1]
```diff
@@ -409,21 +456,68 @@ public static function get_connection_field_by_year( $post_type, $field, $connec
         $start = 0;
         $end = mktime( 24, 60, 60, 12, 31, $current_year );
 
-        $results = $wpdb->get_results(
+        // Determine total number of connections within time frame.
+        $connected = $wpdb->get_results(
             $wpdb->prepare( "
                 SELECT
-                    COUNT( DISTINCT log.object_id ) AS count,
+                    COUNT( DISTINCT p.ID, p.post_title ) AS count,
```
Comment (corsacca) [inline dt-metrics/counters/counter-post-stats.php:463]: @kodinkat, why add p.post_title in the distinct?

Comment (kodinkat) [reply dt-metrics/counters/counter-post-stats.php:463]: @corsacca more so for completeness, but, agreed; we'll be able to get away with just the `p.ID`

Comment (corsacca) [reply dt-metrics/counters/counter-post-stats.php:463]: Lets simplify :)

**Comment by corsacca** [comment]
THanks @kodinkat !

**Pull request merged**